client/jetpack-cloud/sections/agency-signup/agency-signup-form/style.scss

```diff
@@ -0,0 +1,24 @@
+.agency-signup-form {
+	padding: 56px 68px 68px;
+
+	&__logo {
+		display: block;
+		margin-bottom: 52px;
+	}
+
+	&__heading {
+		margin: 0 0 24px;
+		font-weight: 700;
+		font-size: 2.25rem;
+		line-height: 40px;
+		color: #000000;
+	}
+
+	&__subheading {
+		margin: 0 0 38px;
+		font-weight: 400;
+		font-size: 1rem;
+		line-height: 24px;
+		color: #000000;
```

Comment:
We could replace `#000#` with `var( --studio-black );` but I dno how strict we are with it 😅